framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/RouterUtil.java

```diff
@@ -60,8 +60,13 @@ public static void waitForRouter(URL url) {
 
                 Thread.sleep(ROUTER_WAIT_ITERATION_TIME);
             } catch (SSLHandshakeException e) {
-                logger.warn("SSLHandshakeException: " + e.getMessage());
-                logger.info("Try to execute request again.");
+                logger.debug("SSLHandshakeException: " + e.getMessage());
+                logger.debug("Wait for a while and try to execute request again.");
+                try {
+                    Thread.sleep(Duration.ofSeconds(1).toMillis());
+                } catch (InterruptedException e1) {
+                    logger.debug("Exception in thread sleep.", e1);
```

Comment:
Please handle InterruptedException properly, it is thrown usually in case JVM shuts down, see https://github.com/kiegroup/kie-cloud-tests/blob/6e827617a9996f702514f9e8faaa6e8d1db44268/framework-cloud/framework-cloud-common/src/main/java/org/kie/cloud/common/provider/KieServerControllerClientProvider.java#L55